### ISSUE TITLE
Linux Setup Instructions incomplete 

### ISSUE THREAD

**Issue by colinsauze** [swcarpentry/styles#553]
The Linux Setup instructions (in setup.md) miss several steps that are required (at least for me on Ubuntu 18.04) to get the Gems configured and installed. 

After running `gem install bundler --user-install` I get the error:

```
Fetching: bundler-2.2.12.gem (100%)
WARNING:  You don't have /home/colin/.gem/ruby/2.5.0/bin in your PATH,
          gem executables will not run.
Successfully installed bundler-2.2.12
Parsing documentation for bundler-2.2.12
Installing ri documentation for bundler-2.2.12
Done installing documentation for bundler after 3 seconds
1 gem installed
```
so after running this the bundle(r) command is not in my path. I either need to add ~/.gem/ruby/2.5.0/bin to my path or prefix all commands with it.

When trying to run `bundle update` 

I get the error:

```
Fetching gem metadata from https://rubygems.org/...........
Resolving dependencies.....
Following files may not be writable, so sudo is needed:
  /usr/local/bin                                                                                                                                                                                      
  /var/lib/gems/2.5.0                                                                                                                                                                                 
Fetching concurrent-ruby 1.1.8
Fetching zeitwerk 2.4.2
Fetching minitest 5.14.4
Fetching thread_safe 0.3.6


Your user account isn't allowed to install to the system RubyGems.
  You can cancel this installation and run:

      bundle config set --local path 'vendor/bundle'
      bundle install

  to install the gems into ./vendor/bundle/, or you can enter your password
  and install the bundled gems to RubyGems using sudo.

  Password: 
```

This can be solved by running `bundle config set --local path 'vendor/bundle'` and `bundle install` first.

**Comment by maxim-belkin** [comment]
@fmichonneau, I think lesson-example was the right repo for this report... This repo doesn't provide `setup.md`.
The order of `bundle config` flags ~will be fixed in #495~ was fixed in #559.

**Comment by fmichonneau** [comment]
I think that was addressed in the installation instructions in the lesson-example repository.
